crates/db_schema/src/source/images.rs

```diff
@@ -25,7 +25,7 @@ use {
 pub struct LocalImage {
   pub pictrs_alias: String,
   pub published_at: DateTime<Utc>,
-  pub person_id: PersonId,
+  pub person_id: Option<PersonId>,
```

Comment:
Add a note here saying

`/// Before migrations/2025-05-06-145536_local_image_person , non-local thumbnails aren't attached to a person. Afterwards, they are.`

Reply:
Might be useful to clarify other `Option` fields later in other structs.

Reply:
This comment would be part of the api docs where its going to be more confusing than anything. In any case its clear that there is `LocalImage.local_user_id` in 0.19 and `LocalImage.person_id` in 1.0 so the comment is unnecessary.